src/cli.rs

```diff
@@ -324,6 +326,10 @@ impl Subcommand {
     set_term_width = 79
 )]
 pub struct Cli {
+    /// Use configuration found in DIRECTORY
+    #[clap(short, long, value_name = "DIRECTORY")]
+    pub config_dir: Option<PathBuf>,
```

Comment:
Let's change `config_dir` to `config` and omit the short option (for now):

```rust
    #[clap(long, value_name = "DIRECTORY")]
    pub config: Option<std::path::PathBuf>,
```

It may be good to add short options, but I'd like to wait and do them all at once, to make sure that they're compatible with each other (e.g., `config` vs `compression`).